kernel/kernel-impl/src/test/java/org/sakaiproject/content/impl/test/RoleAccessTest.java

```diff
@@ -66,11 +66,17 @@ public void setUp() throws IdUsedException, IdInvalidException, InconsistentExce
         session.setUserId("admin");
 
         _ss = (SiteService)getService(SiteService.class.getName());
-        Site newSite = _ss.addSite(SITE_ID, (String) null);
-        Group group = newSite.addGroup();
-        group.setTitle(".group");
-        _groupReference = group.getReference();
-        _ss.save(newSite);
+        Site newSite = null;
```

Comment:
Just add a second remove call to the tests so it becomes

    // Mark as deleted.
    siteService.remoteSite(site);
    // Remove the deleted site.
    siteService.remoteSite(site);

or add specific config for the test to disable the site recycle bin

    @BeforeClass
	public static void beforeClass() {
		try {
			oneTimeSetup("disable-site-recycle");

and then create a sakai.properties file in the disablesite-recycle folder that disables the site recycle bin. You can see an example of how this is done with the `disable_user_cache` configuration for tests.